sklearn/linear_model/_ridge.py

```diff
@@ -1548,15 +1548,16 @@ def fit(self, X, y, sample_weight=None):
 class _BaseRidgeCV(LinearModel):
     def __init__(self, alphas=(0.1, 1.0, 10.0),
                  fit_intercept=True, normalize=False, scoring=None,
-                 cv=None, gcv_mode=None,
-                 store_cv_values=False):
+                 cv=None, gcv_mode=None, store_cv_values=False,
+                 boundary_warning=False):
```

Comment:
Why do we need a parameter to control the warning? I thought we would want the warning shown by default, and the user can then disable it with the warnings module's filters

Reply:
Sure we can do that as well! I thought we needed a constructor parameter. Can I remove that?

Reply:
I'd be happy to have them filtered with the warning filters, but as suggested by @ogrisel I also think having an easy way to silence the warning is nice to have. Maybe we could add a note in the warning message on how the user can silence it?